### ISSUE TITLE
Gradient Boosting Models

### ISSUE THREAD

**Issue by hongbo-miao** [hongbo-miao/hongbomiao.com#36340]
# Gradient Boosting Models

- https://github.com/dmlc/xgboost
- https://github.com/catboost/catboost
- https://github.com/microsoft/LightGBM
- https://github.com/LinearBoost/linearboost-classifier

## Star History

[![Star History Chart](https://api.star-history.com/svg?repos=dmlc/xgboost,catboost/catboost,microsoft/LightGBM,LinearBoost/linearboost-classifier&type=date&legend=top-left)](https://www.star-history.com/#dmlc/xgboost&catboost/catboost&microsoft/LightGBM&LinearBoost/linearboost-classifier&type=date&legend=top-left)
